### DISCUSSION TITLE
How to set (unset) classes to NavLi?

### DISCUSSION THREAD

**Discussion by alexander-mart** [themesberg/flowbite-svelte#663] [Q&A]
```
<li>
  <a href="/about" class="block py-2 pr-4 pl-3 md:p-0 rounded md:border-0">About</a>
                                               ______
                                               ↑
</li>
```

For example, how to remove `md:p-0` from `<a>` inside of `<NavLi>` component?
It was breaks paddings of menu items on middle screens :(

This is code of component:
https://github.com/themesberg/flowbite-svelte/blob/main/src/lib/navbar/NavLi.svelte#L15

**Comment by shinokada** [comment]
We are hard coding ` 'block py-2 pr-4 pl-3 md:p-0 rounded md:border-0',`.
We can export it in the next version so that you can add your class.

**Comment by alexander-mart** [comment]
:heavy_check_mark: I was overwrite it with `class={'p-2 md:p-2'}`
